pkg/engine/internal/types/column.go

```diff
@@ -44,6 +45,24 @@ func (ct ColumnType) String() string {
 	}
 }
 
+// FromString returns the [ColumnType] from its string representation.
+func (ColumnType) FromString(ct string) ColumnType {
```

Comment:
looking at its current usage `types.ColumnType(0).FromString(ct)`, should this be a separate function and not a method

Reply:
Agree. Should be a function `func ColumnTypeFromString(ct string) ColumnType`